traversals/morris_traversal.py

```diff
@@ -0,0 +1,51 @@
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+
+def Morris_traversal(root):
+    """
+    Morris(InOrder) travaersal is a tree traversal algorithm that does not employ
+     the use of recursion or a stack. In this traversal, links are created as
+    successors and nodes are printed using these links.
+    Finally, the changes are reverted back to restore the original tree.
+    """
+    InorderTraversal = []
```

Comment:
```suggestion
    inorder_traversal = []
```

Reply:
Please do this change wherever the variable was used.

Reply:
Sure, I will do